api/emoji.go

```diff
@@ -252,3 +289,43 @@ func getEmojiImage(c *Context, w http.ResponseWriter, r *http.Request) {
 func getEmojiImagePath(id string) string {
 	return "emoji/" + id + "/image"
 }
+
+func resizeEmoji(img image.Image, width int, height int) image.Image {
+	emojiWidth := float64(width)
+	emojiHeight := float64(height)
+
+	var emoji image.Image
+	if emojiHeight <= MaxEmojiHeight && emojiWidth <= MaxEmojiWidth {
+		emoji = img
+	} else {
+		emoji = resize.Thumbnail(MaxEmojiWidth, MaxEmojiHeight, img, resize.Lanczos3)
```

Comment:
Oh sorry, I think I linked you to the old image library we used to use for resizing when I answered your question a couple weeks ago. Could you change this to use [`imaging.Fit`](https://godoc.org/github.com/disintegration/imaging#Fit) instead? The function call would look basically the same, but it's already used elsewhere so we wouldn't need to include another library.